src/patternfly/patternfly-drag-drop.scss

```diff
@@ -0,0 +1,63 @@
+.pf-c-draggable {
+  --pf-c-draggable--m-dragging--Cursor: grabbing;
```

Comment:
Should you set `--pf-c-draggable--Cursor: grab;` by default, then update to `grabbing`?

Reply:
```suggestion
  --pf-c-draggable--Cursor: grab;
```

Reply:
I wasn't thinking so, unless it's a thing in the draggable element that is only draggable (like the draggable icon). Because I imagine you'd still want your normal cursor for regular stuff in the draggable item (links, text, etc), since that's probably the most common interaction with the draggable item. It would change to dragging once we've detected you probably want to drag the thing. wdyt @redallen